driver/Makefile

```diff
@@ -5,3 +5,6 @@ test:
 
 test_mpi:
 	$(MPIRUN_CALL) python3 -m mpi4py -m pace.driver.run examples/configs/baroclinic_c12.yaml
+
+test_mpi_serialbox:
```

Comment:
Fair enough. Do we expect users to run from make file? Because there're some instructions in driver/README, and they don't reference the makefile either.

Reply:
I don't expect user to run this file soon - the Makefile is here for our development purposes. test_mpi is useful to iterate locally on code without the overhead of compiling/running docker. test_mpi_serialbox could also be useful for certain development, but there's no way to run it other than using `.jenkins/test_driver.sh` since it explicitly depends on being in a docker environment (specifically, having a test_data folder at the filesystem root).

You could get it running automatically through this make target, but I think test_driver.sh should be enough.

Reply:
Okay, `test_mpi_serialbox` removed from Makefile and moved the command directly to `test_driver.sh`. Keeping `baroclinic_c12_from_serialbox.yaml` in the examples/configs folder because it needs to be somewhere docker can access. I think it makes sense for the example yaml file to be there even though it points to a path that people not using docker will not have access, information on the path is added in README.